### ISSUE TITLE
 bundle exec rake db:setup

### ISSUE THREAD

**Issue by NxSolutions** [peatio/peatio#689]
Hi, 
I am trying to run  **bundle exec rake db:setup** command and getting the following error, 
W, [2018-01-07T19:12:25.365925 #10790]  WARN -- : You are setting a key that conflicts with a built-in method OmniAuth::AuthHash::InfoHash#name defined at /root/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/omniauth-1.2.2/lib/omniauth/auth_hash.rb:34. This can cause unexpected behavior when accessing the key as a property. You can still access the key via the #[] method.
/root/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/activesupport-4.0.13/lib/active_support/core_ext/numeric/conversions.rb:121: warning: constant ::Fixnum is deprecated
/root/.rbenv/versions/2.5.0/lib/ruby/gems/2.5.0/gems/activesupport-4.0.13/lib/active_support/core_ext/numeric/conversions.rb:121: warning: constant ::Bignum is deprecated
rake aborted!
SystemStackError: stack level too deep
/root/peatio/config/application.rb:12:in `<top (required)>'
/root/peatio/Rakefile:5:in `require'
/root/peatio/Rakefile:5:in `<top (required)>'
/root/.rbenv/versions/2.5.0/bin/bundle:23:in `load'
/root/.rbenv/versions/2.5.0/bin/bundle:23:in `<main>'
(See full trace by running task with --trace)

Can someone please help me on how to fix it?

Thanks

**Comment by zaidragie** [comment]
I'm getting this error
rake aborted!
Psych::SyntaxError: (): did not find expected key while parsing a block mapping at line 6 column 3
/home/zaid/peatio/config/application.rb:15:in <module:Peatio>' /home/zaid/peatio/config/application.rb:14:in<top (required)>'
/home/zaid/peatio/Rakefile:5:in require' /home/zaid/peatio/Rakefile:5:in<top (required)>'
/home/zaid/.rbenv/versions/2.2.2/bin/bundle:23:in load' /home/zaid/.rbenv/versions/2.2.2/bin/bundle:23:in

**Comment by eric-tsoi** [comment]
Yes, i am getting te same issue too wit ruby 2.5.0. Perhaps needs to downgrade to 2.2.x

**Comment by cyzanfar** [comment]
same here

**Comment by NxSolutions** [comment]
Use different repo as mentioned on  this forum 
https://gitter.im/peatio/peatio

![image](https://user-images.githubusercontent.com/34899085/35214321-ed263ece-ffb4-11e7-9b32-8fd3efbc2c88.png)

**Comment by NxSolutions** [comment]
https://github.com/scatterp/peatio

**Comment by oskanddr** [comment]
Hello @NxSolutions @cyzanfar,

You are getting this error because Peatio's codebase is too old for modern systems. I can help you get Peatio running. I also offer development services.

You can always reach me at nimpo1074@gmail.com

Best,
Oskar

**Comment by algobasket** [comment]
@NxSolutions @zaidragie @eric-tsoi @cyzanfar 
Complete Peatio Help | Support | Bug Fixing
http://peatio.algobasket.com
Free Technical Documentation below :
https://github.com/peatio/peatio/files/2457278/PeatioCryptoExchange.pdf
https://www.youtube.com/watch?v=tpieNIMSBuQ
https://github.com/algobasket/PeatioCryptoExchange/
Contact me on Skype : algobasket
Thanks in advance !!!
